File: Tests/AnnotationsWithMultipleOutputs/FirstType.cs
using KY.Generator;

namespace AnnotationsWithMultipleOutputs
{
    [GenerateAngularModel("Output\\First")]
    [GenerateNoHeader]
    public class FirstType
    {
        public string StringProperty { get; set; }
        public SubType SubTypeProperty { get; set; }
    }
}

File: Tests/AnnotationsWithMultipleOutputs/Output/First/first-type.ts
// tslint:disable

import { SubType } from "./sub-type";

export class FirstType {
    public stringProperty: string;
    public subTypeProperty: SubType;

    public constructor(init: Partial<FirstType> = undefined) {
        Object.assign(this, init);
    }
}

// outputid:352b1947-a770-4737-be18-608a78c130ad